awx/main/tasks.py

```diff
@@ -93,6 +93,12 @@
 logger = logging.getLogger('awx.main.tasks')
 
 
+class InvalidVenvError(Exception):
```

Comment:
One more nitpick - this is going to end up being user-facing, so let's call it `InvalidVirtualenvError`.